wifiphisher/pywifiphisher.py

```diff
@@ -75,6 +75,13 @@ def parse_args():
         "--internetinterface",
         help=("Choose an interface that is connected on the Internet" +
               "Example: -iI ppp0"))
+    parser.add_argument(
+        "-pI",
+        "--protectinterface",
+        nargs='+',
+        help=("Specify the interface(s) that will have their connection protected. " +
```

Comment:
We should clarify here what we mean by "have their connection protected."

Reply:
Something like: "Specify the interface(s) that will have their connection protected (i.e. the tool will prevent Network Manager from controlling them)."